package.json

```diff
@@ -256,7 +256,8 @@
     "lit": "2.8.0",
     "clean-css": "5.3.3",
     "@lit/reactive-element": "1.6.3",
-    "@fullcalendar/daygrid": "6.1.15"
+    "@fullcalendar/daygrid": "6.1.15",
+    "globals": "^13.0.0"
```

Comment:
Why is globals added as a resolution?

Reply:
Older versions have a space mistake where eslint now starts to fail. We still throw in version 11 somewhere.